components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/dao/impl/ApiFileDAOImpl.java

```diff
@@ -0,0 +1,486 @@
+/*
```

Comment:
When using editor app, all the details should be saved in the users local machine's file system. So in order to cater that we need to provide a DAO layer to write to the file system instead of the database. A user can develop apis using editor locallay and these files can be imported to any of the environments later